src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -60,28 +73,35 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
     ])
   }
 
-  const getAppDeletionMessage = () => {
-    return !_.isEmpty(nonDeletableApps) ?
+  const hasApps = () => {
+    return deletableApps !== undefined && nonDeletableApps !== undefined &&
+      (!_.isEmpty(deletableApps) ||
+        !_.isEmpty(nonDeletableApps))
+  }
+
+  const isDeleteDisabledFromResources = (hasApps() && !_.isEmpty(nonDeletableApps)) || controlledResourcesExist
+
+  const getResourceDeletionMessage = () => {
```

Comment:
Made these messages read better since one often only has a single app.

![image](https://user-images.githubusercontent.com/484484/215585752-2feaa5d2-8f65-41a9-9a2b-5a9d8a23da10.png)

![image](https://user-images.githubusercontent.com/484484/215585899-ed47ec5b-d335-4477-8539-cb50f445594c.png)